receiver/hostmetricsreceiver/README.md

```diff
@@ -88,6 +88,15 @@ load:
   cpu_average: <false|true>
 ```
 
+### Memory
+
+`enable_additional_memory_states` specifies whether to enable reporting of a couple of additional memory state metrics: `slab_reclaimable`, `slab_unreclaimable` and `available`. When those are enabled, all the memory states don't sum up well (the result exceeds `total` memory) (default: `false`).
```

Comment:
This isn't a hill I'm going to die on, but in my opinion we should be looking to replace this metric with one or more metrics that use attributes appropriately to differentiate portions of a meaningful total.

I'm not an expert at memory management but it seems someone should be able to clearly state the relationships between these values. As a (probably wrong) starting point:
- `total = buffered + cached + inactive + free + used`
- `slab = slab_reclaimable + slab_unreclaimable`
- `available = free + slab_reclaimable`?

If we had a list of these relationships, it should be fairly easy to identify a correct data model. Without it, I don't know how to evaluate such a solution except to say that overloading a metric has never been necessary elsewhere, as far as I am aware. Again, this isn't intended to be blocking - just trying to articulate _why_ in my opinion we are having to invent novel ways to tweak a metric.

Reply:
Yeah, you're right, we're dealing with a complex issue that doesn't have a simple and universal solution. However, I believe that including the ability to report available memory is a valuable feature for users, despite its potential drawbacks.